decompiler/config/jak2/all-types.gc

```diff
@@ -47055,9 +47148,40 @@
   )
 |#
 
-#|
+(defenum vehicle-controller-flag
+  :type uint32
+  :bitfield #t
+  (do-turn 0)
+  (off-path 1)
+  (attached 2)
+  (no-slowing-for-turns 3)
+  (recovery-mode 4)
+  (blocking-dest-node 5)
+  (direct-mode 6)
+  (ignore-others 7)
+  (debug 8)
+  (on-straightaway 9)
+  (draw-marks 10)
+  (left-turn 11)
```

Comment:
I think this order isn't right - unfortunately the order they are printed in doesn't match the order, so you have to look at the assembly and figure out which bit it's testing.

For example:
```

    lwu s5, 0(gp)             ;; [ 16] (set! s5 (l.wu gp))
    addiu v1, r0, 16          ;; [ 17] (set! v1 16)
    andi a0, s5, 16           ;; [ 18] (set! a0 (logand s5 16))
    bne a0, v1, L53           ;; [ 19] (b! (!= a0 v1) L53 (set! v1 #f))
    or v1, s7, r0

B4:
    lw t9, format(s7)         ;; [ 20] (set! t9 format)
    daddiu a0, s7, #t         ;; [ 21] (set! a0 #t)
    daddiu a1, fp, L225       ;; [ 22] (set! a1 L225) "do-turn "
    jalr ra, t9               ;; [ 23] (call!)
    sll v0, ra, 0

    or v1, v0, r0             ;; [ 24] (set! v1 v0)
 ```
is `and`ing with 16 = `0b10000`, so `do-turn` would be 4.